conf/services/check-transit-guarantee-balance.yml

```diff
@@ -0,0 +1,24 @@
+serviceName: Check your transit guarantee balance
```

Comment:
@jackheslop96 Being picky here, the service name that is rendered in staging is `Check your guarantee balance`, not `transit`. One for Fred?

Reply:
I suspect it's 'transit guarantee balance' because 'guarantee balance' is a bit vague out of context